Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cstr_core"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
I will take care of bumping the version number separately when I publish the crate.